lib/jekyll/configuration.rb

```diff
@@ -54,6 +54,7 @@ class Configuration < Hash
       "permalink"           => "date",
       "paginate_path"       => "/page:num",
       "timezone"            => nil, # use the local timezone
+      "preserve_timezones"  => false, # convert to the above timezone
```

Comment:
I suppose defaulting this to `true` could change permalinks of sites that relied on the old behavior.

Reply:
Yes. People were saying in some older bug reports that they rely on the old behavior. That's why I set the default to `false` and added a line to the default config (`lib/site_template/_config.yml`) to set it to `true` for new sites. Although that is a policy decision; the project might prefer to not do the latter. I don't care either way.

Reply:
I agree with you that this conservative approach is best :+1: